squirrel.worker/src/main/java/org/dice_research/squirrel/analyzer/impl/html/scraper/HtmlScraper.java

```diff
@@ -66,59 +59,19 @@ public HtmlScraper() {
 
     }
 
-    @SuppressWarnings("unchecked")
-	public List<Triple> scrape(String uri, File filetToScrape) throws Exception {
+    public List<Triple> scrape(CrawleableUri curi, File filetToScrape) throws Exception {
 
         List<Triple> listTriples = new ArrayList<Triple>();
         listIterableObjects = new LinkedHashSet<String>();
-
+        uri= curi.getUri().toString();
         YamlFile yamlFile = (YamlFile) yamlFiles.get(UriUtils.getDomainName(uri)).clone();
-        
-        
-        this.uri = uri;
-	     if(uri.contains("?")) {
-	      this.label = uri.substring(uri.lastIndexOf("/")+1, uri.lastIndexOf("?"));
-	     }else {
-	      this.label = uri.substring(uri.lastIndexOf("/")+1, uri.length());
-	     }
```

Comment:
Why did you remove this code block?

Reply:
Added it back. It was missed during resolving merge conflicts.